contracts/src/core/EigenDACertVerifier.sol

```diff
@@ -80,45 +80,45 @@ contract EigenDABlobVerifier is IEigenDABlobVerifier {
     /**
      * @notice Verifies a blob for the base required quorums and the default security thresholds
      * @param batchHeader The batch header of the blob
-     * @param blobVerificationProof The blob verification proof for the blob
+     * @param blobInclusionInfo The inclusion proof for the blob
      * @param nonSignerStakesAndSignature The nonSignerStakesAndSignature for the blob
      */
-    function verifyBlobV2(
+    function verifyDACertV2(
         BatchHeaderV2 calldata batchHeader,
-        BlobVerificationProofV2 calldata blobVerificationProof,
+        BlobInclusionInfo calldata blobInclusionInfo,
         NonSignerStakesAndSignature calldata nonSignerStakesAndSignature
     ) external view {
-        EigenDABlobVerificationUtils._verifyBlobV2ForQuorums(
+        EigenDACertVerificationUtils._verifyDACertV2ForQuorums(
             eigenDAThresholdRegistry,
             eigenDASignatureVerifier,
             eigenDARelayRegistry,
             batchHeader,
-            blobVerificationProof,
+            blobInclusionInfo,
             nonSignerStakesAndSignature,
             getDefaultSecurityThresholdsV2(),
-            blobVerificationProof.blobCertificate.blobHeader.quorumNumbers
+            blobInclusionInfo.blobCertificate.blobHeader.quorumNumbers
         );
     }
 
     /**
      * @notice Verifies a blob for the base required quorums and the default security thresholds
```

Comment:
The `@notice` tag still talks about verifying a "blob", instead of verifying a "cert"

(The other modified functions in this PR also have outdated docs)